pkg/auth/server/grant/templates.go

```diff
@@ -0,0 +1,144 @@
+package grant
+
+import "html/template"
+
+var defaultGrantTemplate = template.Must(template.New("defaultGrantForm").Parse(defaultGrantTemplateString))
+
+const defaultGrantTemplateString = `<!DOCTYPE html>
+
+<html>
+
+<head>
+    <title>Authorize {{ .Values.ClientID }}</title>
+    <style>
+        body    { font-family: sans-serif; line-height: 1.2em; margin: 2em 5%; color: #222; }
+
+        table         { border-collapse: collapse; margin: 2em 0; width: 100%; max-width: 600px; }
+        table caption { text-align: left; }
+        td            { padding: .5em .25em; }
+        tr + tr       { border-top: 1px solid #ddd; }
+
+        h1,h2,h3 { font-weight: normal; line-height: 1.3em; }
+
+        input[type=submit]                      { font-size: 1em;   }
+        input[type=submit] + input[type=submit] { margin-left: 1em; }
+
+        .client-name { color: #888 }
+
+        .scope-title         { font-size: .9em; font-weight: bold; padding-top: .2em; padding-bottom: .25em; }
+        .scope-details       { font-size: .8em; }
+        .redirect-info       { font-size: .8em; margin: 2em 0 2em .25em; }
+        .existing-permission { color: #55a532; text-align: center; }
+
+        .muted   { color: #888;    }
```

Comment:
use something close from the palette https://www.patternfly.org/styles/color-palette/#_

probably `#8b8d8f` for muted and `#cc0000` for error